### ISSUE TITLE
On Events

### ISSUE THREAD

**Issue by matheusml** [asafdav/ng-scrollbar#29]
Hey,
I'd like to know if there's a way to trigger an event when the user reached the bottom of the scroll.
Thanks!

**Comment by TakeshiDaveau** [comment]
I agree with this feature request, furthermor it should be nice if there is an event for the bottom and for the top.

EDIT : 
To add the event when top is reaching you can add these lines 68

```
if(dragger.top === 0) {
    scope.$emit('scrollbar.toTop');
}
if(page.scrollHeight + page.top - page.height === 0) {
    scope.$emit('scrollbar.toBottom');
}
```
